apps/src/code-studio/components/ModelManagerDialog.jsx

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import Button from '@cdo/apps/templates/Button';
+import Applab from '@cdo/apps/applab/applab';
+
+export default class ModelManagerDialog extends React.Component {
```

Comment:
this dialog will live next to and be most similar to these two components, so just linking those here in case it's helpful:
[`<LibraryManagerDialog/>`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx)
[`<AssetManager/>`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/AssetManager.jsx)

The LibraryManagerDialog is newer and more consistent with our newer styles, so i'd go with that one if you're trying to decide how to style something (related to our convo [here](https://codedotorg.slack.com/archives/C016VGH1BT6/p1611258191003100))